statsmodels/regression/_prediction.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Dec 19 11:29:18 2014
+
+Author: Josef Perktold
+License: BSD-3
+
+"""
+
+import numpy as np
+from scipy import stats
+
+# this is similar to ContrastResults after t_test, partially copied and adjusted
+class PredictionResults(object):
+
+    def __init__(self, predicted_mean, var_pred_mean, var_resid,
+                 df=None, dist=None, row_labels=None):
+        self.predicted_mean = predicted_mean
+        self.var_pred_mean = var_pred_mean
```

Comment:
these two names don't sound like following any convention or an understandable generalizable pattern ?

what about the case when we predict other results, for example functions of endog? what are good names?